ls1intum/Artemis
# `Programming exercises`: Fix clickable checkbox on detail view

### Checklist
#### General
#### Client

### Motivation and Context
The pe details page is only for viewing the exercise settings, you should not be able to change them on this page. However, the checkbox for showing test case names to students was still editable here.

### Description
I added the missing `disabled` property.

### Steps for Testing
View the PE details page, see that the checkbox is not clickable.
Edit the PE, view that the checkbox is clickable here.

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots
old:
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/157ef00d-3748-4dbc-849e-ba12f19968e3)

new:
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/38bacec1-abab-486b-ac6d-887d0baca362)
